includes/gateway/class-omise-payment-atome.php

```diff
@@ -103,7 +103,14 @@ private function validateMinRequiredAmount()
         ];
 
         $currency = strtolower(get_woocommerce_currency());
-        $cartTotal = WC()->cart->total;
+        $cart = WC()->cart;
+
+        if ($cart->subtotal === 0) {
```

Comment:
Just curious, why subtotal value zero mean complimentary products?

Reply:
here, complimentary products mean free products. A product for which customer don't have to pay. This does not include shipping cost.

Reply:
Is there any chance `$cart->subtotal` value in float, mean `0.0`. 
Since we are using `===` which mean `0.0 != 0`

Reply:
WooCommerce will send the subtotal as 0, it won't be 0.0. The only case where we get the decimal number is if the amount itself is in float or if we do some calculation.